app/components/Account/AccountVesting.jsx

```diff
@@ -163,7 +164,12 @@ class VestingBalance extends React.Component {
 
         if (vb) {
             balance = vb.balance.amount;
+            if (!balance) return null;
+
             cvbAsset = ChainStore.getAsset(vb.balance.asset_id);
```

Comment:
Wouldn't it be better to use `FetchChainObjects` to fetch all vesting balances at the same time?

Reply:
FetchChainObjects unfortunately also only does single fetches ... don't ask why ...

Reply:
Oh, it just acts as an array parser. Didn't see that :|